### ISSUE TITLE
"acceptTypes": is "types

### ISSUE THREAD

**Issue by domenic** [arv/ecmascript-object-observe#14]

**Comment by arv** [comment]
Allen's concern was that it might be confusing with types (as in number, Array etc).

__types__ is not good because it will make things less clear if we add filtering on property names and other potential features in the future.

These kind of APIs benefit from being overly clear.

**Comment by domenic** [comment]
Right, I didn't mean "is 'types' the right substitute for 'acceptTypes'?" I meant "is 'types' the right word to use when describing this type of thing?"

The only alternatives I remember being mentioned in the meeting were "kind" and "action".
